pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -110,8 +110,8 @@
             return <div ref="terminal"
                         key={this.state.terminal}
                         className="console-ct"
-                        onFocusIn={this.onFocusIn}
-                        onFocusOut={this.onFocusOut} />;
+                        onFocus={this.onFocusIn}
+                        onBlur={this.onFocusOut} />;
```

Comment:
These events don't work with react-lite, the handler is never called. So either both properties should be set for a transitional period, or this is blocked on actually switching to React. At that point the handler methods should be renamed accordingly, too.